lib/elasticband/aggregation.rb

```diff
@@ -93,6 +100,15 @@ def parse_top_hits(root_aggregation, options_top_hits)
         aggregation = Aggregation::TopHits.new(name, size, options.except(*PARSE_AGGREGATIONS))
         parse_aggregations(options, aggregation)
       end
+
+      def parse_filter(aggregation_name, options)
```

Comment:
I'm using splat to maintain the readability on the hash passed to `parse`. I think we can redefine how we call them later, when we have more aggregations, because this is difficult yet